### DISCUSSION TITLE
Tailwind Components - A repository for tailwindcss components

### DISCUSSION THREAD

**Discussion by mnaderian** [tailwindlabs/tailwindcss#2647] [Show and tell]
I have started a project to collect free open source components for Tailwind CSS. You can add your own components with a single PR.  
Any new idea is welcome.

Here's the repo:  https://github.com/mnaderian/tailwind-components

**Comment by GSCrawley** [comment]
Could you stop blowing up my damn phone with these notifications?? I keep
unsubscribing to no avail



On Thu, Oct 22, 2020 at 3:2
-- 
Gideon S Crawley
646-667-5520

**Reply by JNavith** [reply]
Instead of unsubscribing from individual threads, unwatch the `tailwindlabs/tailwindcss` repository as a whole:
![Unwatch button on repository](https://user-images.githubusercontent.com/28162694/96914480-0dc8f380-1473-11eb-9edc-6d0a86d770f3.png)
⬆ Click that "Unwatch" button

**Comment by simonswiss** [comment]
Hey @mnaderian,

While sharing components with the community is a good idea, you should not be copying components from Tailwind UI and sharing them in your own repo.

Sharing your own creations is totally fine and actually encouraged, but please don't redistribute stuff that clearly comes from Tailwind UI in your own repo, like the sidebar layout and card examples below

![image](https://user-images.githubusercontent.com/485747/96942532-242d8880-1521-11eb-8601-92c1a6fab908.png)

![image](https://user-images.githubusercontent.com/485747/96942595-5343fa00-1521-11eb-9096-de479316437f.png)

Thanks for your understanding 👍

**Comment by mnaderian** [comment]
Thank you @simonswiss for your answer. I don't mean any copyright infringment. These components are just remakes and not copys. I have coded them from scratch actually.

But if remaking something specially from TailwindUI would mean against the community, I would consider that and remove these components. Thanks again.

**Reply by simonswiss** [reply]
No problem, thanks for understanding. 👍
